05-interactive-dst.md

```diff
@@ -192,10 +192,10 @@ functors](06-loki-functors.html)!
 > For example, to explore the `DST` we could have simply done:
 >
 > ```
-> lb-run Bender bender 00035742_00000002_1.allstreams.dst
+> lb-run Bender/latest bender 00035742_00000002_1.allstreams.dst
```

Comment:
Actually `Project/version` works always (I forgot to update the --help message when I added the feature to `lb-run`).

The syntax with a space works only for "standard" versions (vXrY, latest), but not with more original versions (master, upgradeTracking) that you may find in the nightly builds.

At some point I'll drop the syntax with a space... in a far future.

About this specific line, there's a difference between `lb-run` and  `SetupProject`: when you do not specify the version `SetupProject` uses _latest_, while `lb-run` uses _prod_ which is almost never defined (in which case `lb-run` falls back on `SetupProject`). What is really wanted here is `lb-run Bender/latest <command>` and not `lb-run Bender <command>`.

One more note: LHCbDirac does have a defined _prod_ version, so `lb-run LHCbDirac <command>` works correctly.